chainer/functions/lstm.py

```diff
@@ -179,5 +190,21 @@ def lstm(c_prev, x):
     `Long Short-Term Memory in Recurrent Neural Networks \
     <http://www.felixgers.de/papers/phd.pdf>`_.
 
+    .. admonition:: Example
+
+        Assuming ``y`` is current input signal, ``c`` is previous cell state,
+        and ``h`` is previous output signal from ``lstm`` layer with all the
+        length are ``n_units``. Most typical prepareation of ``x`` is:
+
+        >>> model = FunctionSet(w=F.Linear(n_units, 4 * n_units),
+        ...                     v=F.Linear(n_units, 4 * n_units),
+        ...                     ...)
+        >>> x = model.w(y) + model.v(h)
+        >>> c, h = F.lstm(c, x)
+
+        It corresponts to calcurate each input sources :math:`a, i, f, o` using
+        different weight matrices and biases for current input ``y``
+        and previous output ``h``.
```

Comment:
-> It **corresponds** to **calculate** **the** input sources :math:`a, i, f, o` **from** **the** current input ``y`` and **the** previous output ``h``. **Different parameters are used for different kinds of input sources.**